### DISCUSSION TITLE
cn/2024/08/07/false-participation/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#170] [Announcements]
# cn/2024/08/07/false-participation/

https://hongtaoh.com/cn/2024/08/07/false-participation/

**Comment by Finderaha** [comment]
凡事过犹不及，适当的从目标和执行中解脱出来也挺好的。
我们被教育要按时吃饭，不能饿肚子。却很少人提应该定时补充精神能量。

**Reply by hongtaoh** [reply]
有道理。确实，凡事过度都不好。要保持平衡。
